src/Pages/index.js

```diff
@@ -0,0 +1,56 @@
+import {
+  city,
+  Temp,
+  Description,
+  Humidity,
+  windSpeed,
+  Icon,
+  searchBar,
+  searchButton,
+} from "./Dom-module/domPage.js";
+
+let weather = {
+  apiKey: "Add Api Key Here",
+  fetchWeather: function (city) {
+    fetch(
+      "https://api.openweathermap.org/data/2.5/weather?q=" +
+        city +
+        "&units=metric&appid=" +
+        this.apiKey
+    )
+      .then((response) => response.json())
+      .then((data) => this.displayWeather(data));
+  },
+
+  displayWeather: function (data) {
+    const { name } = data;
+    const { icon, description } = data.weather[0];
+    const { temp, humidity } = data.main;
+    const { speed } = data.wind;
+
+    city.innerText = "Weather in " + name;
+    Icon.src = "https://openweathermap.org/img/wn/" + icon + ".png";
```

Comment:
use template literals for concatenating variables with strings. 

As you already used template literals in your previous PR, you didn't use them here, if you aren't following the good practices you already learned, then what's the point?